x-pack/plugins/security_solution/public/detections/components/endpoint_responder/use_responder_action_data.ts

```diff
@@ -160,10 +160,10 @@ export const useResponderActionData = ({
         platform: agentInfoFromAlert.host.os.family,
       });
     }
-    if (hostInfo) {
+    if (isEndpointHost && hostInfo) {
```

Comment:
`isEndpointHost` check is redundant but narrows the `agentType` to `endpoint` so good to have here.